src/metis/http/gist.lua

```diff
@@ -0,0 +1,285 @@
+--- gist.lua - Gist client for ComputerCraft
+-- Made by JackMacWindows for CraftOS-PC and CC: Tweaked
+--
+-- @module cc.http.gist
+
+local expect = require and require("cc.expect").expect or dofile("/rom/modules/main/cc/expect.lua").expect
+
+if not http then
+    if _G.config ~= nil then error("Gist requires http API\nSet http_enable to true in the CraftOS-PC configuration")
+    else error("Gist requires http API\nSet http_enable to true in ComputerCraft's configuration") end
```

Comment:
:sparkles: The config option has not been called `http_enable` for a long time now. I'd just say "enable HTTP in ComputerCraft's config", to avoid having to keep them in sync.